cypress/integration/dragging.spec.ts

```diff
@@ -91,6 +93,82 @@ describe('Dragging', () => {
           .dispatch('mouseup');
       });
     });
+
+    describe('selection with dragOverItems set to false', () => {
+      it('should not start selection over items', () => {
+        disableDragOverItems().then(() => {
+          getDesktopExample().within(() => {
+            cy.getSelectItem(0)
+              .dispatch('mousedown', { button: 0 })
+              .getSelectItem(6, 'end')
+              .dispatch('mousemove')
+              .shouldSelect([1])
+              .getSelectBox()
+              .then(shouldBeInvisible)
+              .get('@end')
+              .dispatch('mouseup');
+          });
+        });
+      });
+
+      it('should start selection in element inbetween SelectContainer and SelectItem', () => {
+        disableDragOverItems().then(() => {
+          getDesktopExample().within(() => {
+            cy.get('mat-grid-list')
+              .as('end')
+              .scrollIntoView()
+              .wait(16)
```

Comment:
Why are all these manual wait points necessary? Generally I'd say it's not ideal especially because Cypress has automatic wait points built-in in almost all APIs. Can you elaborate why these are needed?

Reply:
Just removed the waits and it's working just fine... not sure if these were required back when I created the PR but I probably just copypasta'd from https://github.com/d3lm/ngx-drag-to-select/blob/75a05d48cbf18814430d476e161230dce580cd01/cypress/support/commands.ts#L14 without really looking/understanding the above comment. So I guess it's not necessary for my test scenario?

Plus I'm not familiar with Cypress, only been using Playwright so far 🙈